klab.hub/src/main/java/org/integratedmodelling/klab/hub/tokens/services/RegistrationTokenServiceImpl.java

```diff
@@ -34,6 +35,17 @@ public TokenClickback createToken(String username, TokenType type) {
 			return new CreateChangePasswordToken(repository, username, linkConfig).execute();
 		} else if(type.equals(TokenType.lostPassword)) {
 			return new CreateLostPasswordToken(repository, username, linkConfig).execute();
+		} else if(type.equals(TokenType.verifyEmail)) {
+			return new CreateVerifyEmailToken(repository, username, linkConfig).execute();
+		} else {
+			return null;
```

Comment:
Do we want to return `null` or throw an Exception? 🤔

Reply:
Is for keeping the same structure.